<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="d8b3f382-7fbe-49a5-b421-4c48e4e34c91" name="Paolo Possanzini@ARCHERAIR 2012-04-27 15:36:35" runUser="ArcherAir\Paolo Possanzini" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="b64f6e3c-f874-4d40-80bf-369ba9bd7b4f">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Paolo Possanzini_ARCHERAIR 2012-04-27 15_36_35">
      <DeploymentItem filename="C:\temp\Pegaso 2000\MultiThreading\ParallelSearch\bin\Debug\ParallelSearch.exe" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2012-04-27T15:36:35.7138672+02:00" queuing="2012-04-27T15:36:35.9951172+02:00" start="2012-04-27T15:36:36.0576172+02:00" finish="2012-04-27T15:36:36.7919922+02:00" />
  <ResultSummary outcome="Error">
    <Counters total="1" executed="1" error="1" failed="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
    <RunInfos>
      <RunInfo computerName="ARCHERAIR" outcome="Error" timestamp="2012-04-27T15:36:36.3544922+02:00">
        <Text>One of the background threads threw exception: 
System.NullReferenceException: Object reference not set to an instance of an object.
   at ParallelSearch.Program.&lt;&gt;c__DisplayClass4`1.&lt;ParallelSearch&gt;b__2(Object mymutex) in C:\temp\Pegaso 2000\MultiThreading\ParallelSearch\Program.cs:line 57
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.runTryCode(Object userData)
   at System.Runtime.CompilerServices.RuntimeHelpers.ExecuteCodeWithGuaranteedCleanup(TryCode code, CleanupCode backoutCode, Object userData)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean ignoreSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart(Object obj)</Text>
      </RunInfo>
      <RunInfo computerName="ARCHERAIR" outcome="Error" timestamp="2012-04-27T15:36:36.3701172+02:00">
        <Text>One of the background threads threw exception: 
System.NullReferenceException: Object reference not set to an instance of an object.
   at ParallelSearch.Program.&lt;&gt;c__DisplayClass4`1.&lt;ParallelSearch&gt;b__2(Object mymutex) in C:\temp\Pegaso 2000\MultiThreading\ParallelSearch\Program.cs:line 57
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.runTryCode(Object userData)
   at System.Runtime.CompilerServices.RuntimeHelpers.ExecuteCodeWithGuaranteedCleanup(TryCode code, CleanupCode backoutCode, Object userData)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean ignoreSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart(Object obj)</Text>
      </RunInfo>
      <RunInfo computerName="ARCHERAIR" outcome="Error" timestamp="2012-04-27T15:36:36.3701172+02:00">
        <Text>One of the background threads threw exception: 
System.NullReferenceException: Object reference not set to an instance of an object.
   at ParallelSearch.Program.&lt;&gt;c__DisplayClass4`1.&lt;ParallelSearch&gt;b__2(Object mymutex) in C:\temp\Pegaso 2000\MultiThreading\ParallelSearch\Program.cs:line 57
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.runTryCode(Object userData)
   at System.Runtime.CompilerServices.RuntimeHelpers.ExecuteCodeWithGuaranteedCleanup(TryCode code, CleanupCode backoutCode, Object userData)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean ignoreSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart(Object obj)</Text>
      </RunInfo>
      <RunInfo computerName="ARCHERAIR" outcome="Error" timestamp="2012-04-27T15:36:36.4169922+02:00">
        <Text>One of the background threads threw exception: 
System.NullReferenceException: Object reference not set to an instance of an object.
   at ParallelSearch.Program.&lt;&gt;c__DisplayClass4`1.&lt;ParallelSearch&gt;b__2(Object mymutex) in C:\temp\Pegaso 2000\MultiThreading\ParallelSearch\Program.cs:line 57
   at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
   at System.Threading.ExecutionContext.runTryCode(Object userData)
   at System.Runtime.CompilerServices.RuntimeHelpers.ExecuteCodeWithGuaranteedCleanup(TryCode code, CleanupCode backoutCode, Object userData)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean ignoreSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.ThreadHelper.ThreadStart(Object obj)</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="ListaConPochiValori" storage="c:\temp\pegaso 2000\multithreading\testproject1\bin\debug\testproject1.dll" id="915ecda9-bde9-57f2-ada3-100f4f9eaa2f">
      <Execution id="36bd3eed-492f-4d46-8c0f-8c2b912ded0f" />
      <TestMethod codeBase="C:/temp/Pegaso 2000/MultiThreading/TestProject1/bin/Debug/TestProject1.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="TestProject1.UnitTest1, TestProject1, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="ListaConPochiValori" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="915ecda9-bde9-57f2-ada3-100f4f9eaa2f" executionId="36bd3eed-492f-4d46-8c0f-8c2b912ded0f" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="36bd3eed-492f-4d46-8c0f-8c2b912ded0f" testId="915ecda9-bde9-57f2-ada3-100f4f9eaa2f" testName="ListaConPochiValori" computerName="ARCHERAIR" startTime="2012-04-27T15:36:36.0732422+02:00" endTime="2012-04-27T15:36:36.7919922+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="36bd3eed-492f-4d46-8c0f-8c2b912ded0f">
      <Output>
        <ErrorInfo>
          <Message>The agent process was stopped while the test was running.</Message>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>